src/hotspot/cpu/ppc/templateTable_ppc_64.cpp

```diff
@@ -3327,7 +3395,7 @@ void TemplateTable::prepare_invoke(int byte_no,
   // Determine flags.
   const Bytecodes::Code code = bytecode();
   const bool is_invokeinterface  = code == Bytecodes::_invokeinterface;
-  const bool is_invokedynamic    = code == Bytecodes::_invokedynamic;
+  const bool is_invokedynamic    = code == false; // should not reach here with invokedynamic
```

Comment:
This looks strange! I guess you wanted to delete more?

Reply:
Basically I kept the local variable as a name for the (now) constant value passed in the call at L3409.

The parameter cannot be eliminated since `load_invoke_cp_cache_entry()` is declared in a shared header.

I could replace the variable reference in the call with `false /* is_invokedynamic */` if you like that better. Personally I'd prefer the current version.

Reply:
I meant `code == false`. That was probably not intended.

Reply:
Oh my ... Your are right of course.